### ISSUE TITLE
False positive for too-many-function-args with numpy.datatime64.astype

### ISSUE THREAD

**Issue by insysion** [PyCQA/pylint#3332]
### Steps to reproduce
1. `pip install requirements.txt` [(requirements.txt)](https://github.com/PyCQA/pylint/files/4031633/requirements.txt)
```
astroid==2.3.3
colorama==0.4.3; sys_platform == "win32"
isort==4.3.21
lazy-object-proxy==1.4.3
mccabe==0.6.1
numpy==1.18.1
pylint==2.4.4
six==1.13.0
typed-ast==1.4.0; implementation_name == "cpython" and python_version < "3.8"
wrapt==1.11.2
```
2.  `pylint bug.txt` [(bug.txt)](https://github.com/PyCQA/pylint/files/4031639/bug.txt)
```
"""Bug"""

import datetime

import numpy as np

NP_DT64 = np.datetime64(1, 'us')
NP_DT64.astype(datetime.datetime)
```

### Current behavior
```
************* Module bug
tmp.py:8:0: E1121: Too many positional arguments for method call (too-many-function-args)

--------------------------------------------------------------------
```

### Expected behavior

-------------------------------------------------------------------

### pylint --version output
```
pylint 2.4.4
astroid 2.3.3
Python 3.7.3 (default, Apr  3 2019, 19:16:38) 
[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]
```
